packages/gestalt/src/Tooltip/InternalTooltip.tsx

```diff
@@ -166,7 +169,7 @@ export default function InternalTooltip({
               padding={2}
               tabIndex={0}
             >
-              <Text color="inverse" size="100">
+              <Text color="inverse" dataTestId={dataTestId} size="100">
```

Comment:
why is  dataTestId={dataTestId} instead of dataTestId={`${data-test-id}-text`}?

im not saying it's one way or another, but we ended not agreeing on a pattern for a component anatomy.

The pattern could:
- we append the "element" that it's added to
- we dont append the "element" to the main element in the component. Foer example, in textField we don't append "input"

My thinking is that, how we communicate that the datatestid is going to the test instead of the floating container?